Do not color test output as it messes up the terminal

Index: tests/testlib.lua
--- tests/testlib.lua.orig
+++ tests/testlib.lua
@@ -2,9 +2,9 @@
 posix = require( 'posix' )
 
 -- escape codes to colorize output on terminal
-local c1='\027[47;34m'
+local c1=''
 
-local c0='\027[0m'
+local c0=''
 
 --
 -- Writes colorized.
@@ -34,7 +34,7 @@ cwriteln( 'random seed: ', seed )
 --
 function mktempd
 ( )
-	local f = io.popen( 'mktemp -td ltest.XXX', 'r' )
+        local f = io.popen( 'mktemp -td ltest.XXXXXXXX', 'r' )
 
 	local s = f:read( '*a' )
 
